src/utils/__tests__/style-formatter.spec.js

```diff
@@ -96,25 +97,19 @@ describe('style-formatter', () => {
           },
         },
       };
-      const formattedStyle = styleFormatter.getStyles(style, Theme);
+      const formattedStyle = styleFormatter.getStyles(style, disabled, Theme);
       expect(formattedStyle.includes('background-position: 50% 50%')).to.be.true;
     });
 
     it('should have set opacity for disabled button', () => {
-      style = {
-        useEnabledCondition: true,
-        enabledCondition: 0
-      };
-      const formattedStyle = styleFormatter.getStyles(style, Theme);
+      disabled = true;
+      const formattedStyle = styleFormatter.getStyles(style, disabled, Theme);
```

Comment:
You could set true inside the function here as well. Then you don't have to reset disabled in the next test

Reply:
like this?
`const formattedStyle = styleFormatter.getStyles(style, true, Theme);`

Reply:
exactly! In that way you don't change the default values and start every unit test in the same way. Another way would be to have a beforeEach function, but that seems a bit much for this single test